Source/Cli/cli.cpp

```diff
@@ -109,6 +203,16 @@ int Cli::exec(QCoreApplication &a)
                 << "-a" << std::endl
                 << "    All (stats + thumbnails + panels)." << std::endl
                 << "    Is default." << std::endl
+                << "-cd <cache directory path>" << std::endl
```

Comment:
I'm wondering if `cache directory` is a good name or if `report directory` would be more clear.

Reply:
@bturkus likes QCvault